src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+function numberWithCommas(number) {
+  return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+}
+
+const populationsDocuments = document.querySelectorAll('.population');
+const totalPopulationDocument = document.querySelector('.total-population');
+const avaragePopulationDocument = document.querySelector('.average-population');
```

Comment:
What is Document in variable name? It can be just populationList, totalPopulation, ...